modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -558,7 +558,40 @@ As such, ensure that `dbms.cluster.discovery.version=V2_ONLY`, and that `dbms.cl
 are set as required, so that the servers start with v2 running on the next restart.
 ====
 
+[[discovery-v1-to-v2-semi-automated-procedure]]
+=== Semi-automated procedure
 
+Another way to migrate from Discovery Service `V1` to `V2` is to use a semi-automated procedure, which simplifies the migration process
+It automates the approach described in the xref:clustering/setup/discovery.adoc#discovery-v1-to-v2-procedures[previous section].
+
+. Make sure that server side routing is enabled as detailed in xref:clustering/setup/routing.adoc#clustering-routing[Server-side routing] section
+. For all the servers, ensure that new settings are updated to the configuration as detailed the xref:clustering/setup/discovery.adoc#_preparation[Preparation] section.
+. Connect to any server using server-side routing
++
+[source,cypher]
+----
+./cypher-shell -a neo4j://localhost:7691 -d system
+----
+
+. Run the following procedure three times to move all members from `V1_ONLY` to `V2_ONLY`. If the procedure fails, read the error message and resolve the issue manually.
+One possible reason for failure is that some members might have had inconsistent `dbms.cluster.discovery.version` settings initially.
+Refer to the previous section for guidance on debugging and fixing issues.
++
+[source,cypher]
+----
+CALL internal.dbms.cluster.moveToNextDiscoveryVersion();
+----
+. To check that the migration was successful, you can execute `CALL dbms.cluster.showParallelDiscoveryState()` on all members of the cluster. You should see that V2_ONLY is the current version.
++
+[queryresult]
+----
++-------------------------------------------------------------+
+| mode      | stateComparison | v1ServerCount | v2ServerCount |
++-------------------------------------------------------------+
+| "V2_ONLY" | "N/A"           | "N/A"         | "3"           |
++-------------------------------------------------------------+
+----
+. In the last step, set the `dbms.cluster.discovery.version=V2_ONLY` for each member of the cluster, so that the version change is retained after a restart.
```

Comment:
We need more detail on what 'set' means here, i.e.
```suggestion
. In the last step, add `dbms.cluster.discovery.version=V2_ONLY` to the config file for each member of the cluster, so that the version change is retained after a restart.
```